### ISSUE TITLE
Bonechewer Raider's Wolf in Hellfire despawns when it shouldn't

### ISSUE THREAD

**Issue by Oldwal** [ccshiro/cc-buglist#73]
The Bonechewer Raider mobs in Hellfire Peninsula spawn a wolf once attacked. What i noticed is that the wolf will despawn once the Bonechewer Raider is killed.

Comments made about this mob on wowhead:
"By Delvara (7,473 – 1·10·72) on 2009/06/15 (Patch 3.1.3)  
What I do is kill off him and then Shadowmeld, the wolf despawns."

So my guess is that the wolf should only despawn if it has nothing to attack (shadowmeld, vanish, feign death), not when the Bonechewer Raider dies as it is now.

Also, something i could not test (was on a Priest) was if he tried to spawn the wolf if he was aggroed with some kind of CC. 

Read the following comment:
"By entropicdragon (51,883 – 12·58·360) on 2007/06/04 (Patch 2.1.0)  
Pulling the Raider with any incapacitating spell or ability will prevent him from spawning his wolf, making the fight a fair bit easier.

Spells which have been verified to prevent it include a Mage's polymorph (you can immediately break it), and any stunning ability (Pounce, Charge, Cheap Shot, etc).
Presumably pulling with other mechanics (fear, charm, silence, etc) will also prevent the spawn, but I haven't seen verification of this (please comment!)"

So basically, the Bonechewer Raiders should only try to summon the wolf once (when aggroed) and the wolf should only despawn if it has no target to attack.

**Comment by Oldwal** [comment]
This is one of the first mobs that you run into when starting on the server. Might wanna get this fixed since first impressions of a server is quite important.

**Comment by YourNewfriend** [comment]
https://github.com/ccshiro/cc-buglist/issues/1058
